iroh-net-report/src/reportgen.rs

```diff
@@ -898,8 +896,15 @@ async fn run_quic_probe(
         ProbeProto::QuicIpv6 => debug_assert!(relay_addr.is_ipv6()),
         _ => debug_assert!(false, "wrong probe"),
     }
-    // TODO(ramfox): what to put here if no host is given?
-    let host = url.host_str().unwrap_or("localhost");
+    let host = match url.host_str() {
+        Some(host) => host,
+        None => {
+            return Err(ProbeError::Error(
```

Comment:
Hum, I believe this should be `ProbeError::AbortSet`?  Why did you choose just this one?  If the RelayUrl has no host none of the retries after a small delay will work either.